packages/postgres-database/src/simcore_postgres_database/models/projects_metadata.py

```diff
@@ -47,10 +48,29 @@
         server_default=sa.text("'{}'::jsonb"),
         doc="Reserved for the user to store custom metadata",
     ),
+    sa.Column(
+        "parent_project_uuid",
```

Comment:
Very nice. I am wondering if it would make sense to move these to a different table. I think there will be a lot more of this type of metadata coming from the metamodeling. But OK, we can also do that at a later point.

Reply:
still not sure also. but we can move these later.